### PR TITLE
add declaration file for 'Bumblebee hotword'

### PR DESCRIPTION
**Pull request by sheunglaili**
- [ ✅ ] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ✅  ] Test the change in your own code. (Compile and run.)
- [ ✅  ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ✅  ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ✅  ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ✅  ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [✅  ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ✅ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ✅  ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ✅ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ✅ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ✅ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## bumblebee-hotword ([<kbd>unpkg</kbd>](https://unpkg.com/browse/bumblebee-hotword@latest/))
was missing the following properties:
1. SpectrumAnalyser
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8df9ca9938b044256351fcf1d8a159ada7f23c45
</p>

File: types/bumblebee-hotword/bumblebee-hotword-tests.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import Bumblebee , { VoiceProcessor , SpectrumAnalyser } from 'bumblebee-hotword';
```
Comment (amcasey) [inline types/bumblebee-hotword/bumblebee-hotword-tests.ts:1]: Personally, I like to match the import style in the sample code in the project readme: `const BumbleBee = require('bumblebee-hotword');`.

**Review by amcasey — APPROVED** [review]

File: types/bumblebee-hotword/bumblebee-hotword-tests.ts (hunk 1 above)
Comment (sheunglaili) [reply types/bumblebee-hotword/bumblebee-hotword-tests.ts:1]: I can make a changes to that.

Comment (sheunglaili) [reply types/bumblebee-hotword/bumblebee-hotword-tests.ts:1]: but seems tslint will complain about that.

**Comment by sheunglaili** [comment]
Ready to merge

**Pull request merged**